# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1455005049 18000
#      Tue Feb 09 03:04:09 2016 -0500
# Node ID 721539013db4f9a09b9589412e935a4bd7d3358f
# Parent  a83e7a384ee0b9c8cbd8313d798c8e4751016a8a
make gl2ps printing work again (bug #47080)

* gl2ps-print.cc (gl2ps_print): Pass stream to gl2ps renderer.

diff --git a/libinterp/corefcn/gl2ps-print.cc b/libinterp/corefcn/gl2ps-print.cc
--- a/libinterp/corefcn/gl2ps-print.cc
+++ b/libinterp/corefcn/gl2ps-print.cc
@@ -778,17 +778,17 @@ gl2ps_print (const graphics_object& fig,
       if (! fp)
         error ("gl2ps_print: failed to create file \"%s\"", stream.c_str ());
 
       frame.add_fcn (safe_fclose, fp);
     }
 
   gl2ps_renderer rend (fp, term);
 
-  rend.draw (fig, "");
+  rend.draw (fig, stream);
 
   // Make sure buffered commands are finished!!!
   rend.finish ();
 
 #else
 
   err_disabled_feature ("gl2ps_print", "gl2ps");
 
